### ISSUE TITLE
import SQLite3 produces error

### ISSUE THREAD

**Issue by MkeC4128** [UPBGE/upbge#1005]
When I try import SQLite3 in the python console or a script I get the following error:

`Traceback (most recent call last):
  File "<blender_console>", line 1, in <module>
  File "C:\...\Release\2.79\python\lib\sqlite3\__init__.py", line 23, in <module>
    from sqlite3.dbapi2 import *
  File "C:\...\Release\2.79\python\lib\sqlite3\dbapi2.py", line 27, in <module>
    from _sqlite3 import *
ImportError: DLL load failed: The specified module could not be found.`

This is in UPBGE 0.2.4.   Note this works fine in Blender 2.79

**Comment by marechal-p** [comment]
OS?
How did you install UPBGE?
How did you install sqlite3?

**Comment by MkeC4128** [comment]
marechal-p I installed the latest stable release of UPBGE 0.2.4b 64 bit (October 14, 2018).  I downloaded it again and also the latest release and the error result.

I just out of curiosity I tried UPBGE 0.2.2b and 0.2.3b and they both worked fine!  It's like there is a bad reference somehow but that doesn't quite make sense either.  I'll try your dll's and let you know.

joelgomes1994, there is no sqlite3.dll in the lib directory of any version of UPBGE or blender.
